static/elements/chromedash-feature.js

```diff
@@ -178,7 +179,7 @@ class ChromedashFeature extends LitElement {
     this._fireEvent('filter-component', {val: e.currentTarget.innerText});
   }
 
-  subscribeToFeature(e) {
+  starFeature(e) {
```

Comment:
nit: since it's staring and un-staring. rename it to "toggleStar"?

Reply:
Done.